receiver/sqlserverreceiver/testdata/config.yaml

```diff
@@ -1,2 +1,12 @@
 sqlserver:
   collection_interval: 10s
+
+sqlserver2:
```

Comment:
```suggestion
sqlserver/named:
```

This isn't a valid component ID unless it follows `type[/name]` format.

Reply:
updated component ID